.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+*     @googleapis/cloudevents-admins
```

Comment:
I don't think cloudevents-admins is likely to be appropriate here. PRs are likely to be either just regeneration of code, or changing manually written C#.

In this case, yoshi-dotnet would be fine - but I wonder whether we should have equivalent teams for "serverless" in general, on a per-language basis.

Reply:
Happy to go either way!  How about I make it `yoshi-dotnet` for now, and then if we come back around with serverless specific teams we change it then?

Reply:
SGTM. That may not be appropriate for all languages, of course...

Reply:
Updated! PTAL